app/memberDetailTemp.html

```diff
@@ -0,0 +1,114 @@
+<html ng-app = "teamform-memberDetail-app">
+<head> 
+<title>Profile testing</title> 
+
+      <!-- Customized libraries shared by pages -->
+      <script src="js/site.js"></script>
+</head>
+<body>
+
+<div id = "YourProfile" ng-controller = "ProfileAcc" class="text-center">
+   Please input your data:)<br>
+	<form id = "ProfileForm">
+		<label>Your UserId:<input type=text ng-model = "FUID"></label><br>
+		<label>Your Name:<input type=text ng-model = "Fname"></label><br>
+		<label>You are well-known in :<input type=text ng-model = "Fskill"></label><br>
+		<label>You apply for activity: <input type = text ng-model = "activity"></label><br>
+		<input type = "button" ng-click = "load()" value = "load"> <input type = "button" ng-click = "save()" value = "save">
+	</form>
+	<div id = "testing_ouput">
+		{{ datadebug | json}}
+	</div>
+   
+   </div>
+   
+      <!-- Angular and Angular-route libraries (v1.5.7) -->
+   <script src="lib/angular.min.js"></script>
+   <script src="lib/angular-route.min.js"></script>
+   
+      <!-- Firebase (v3.0.3) and AngularFire (v2.0.0) (after angular library is loaded) -->
+   <script src="lib/firebase.js"></script>     
+   <script src="lib/angularfire.min.js"></script>
+
+     <script>
+	 
+angular.module('teamform-memberDetail-app',['firebase'])
+.controller('ProfileAcc',  ['$scope', '$firebaseObject', '$firebaseArray',function($scope) {
+	
+	//initalizeFirebase();
+	initalizeFirebase();
+	database = firebase.database();
+	
+	$scope.FUID = "001";
```

Comment:
because just use it as the autofill, changing it to none is better or changing it to login info is better :)